app/grandchallenge/workstation_configs/forms.py

```diff
@@ -0,0 +1,10 @@
+from django.forms import ModelForm
+
+from grandchallenge.core.forms import SaveFormInitMixin
+from grandchallenge.workstation_configs.models import WorkstationConfig
+
+
+class WorkstationConfigForm(SaveFormInitMixin, ModelForm):
+    class Meta:
+        model = WorkstationConfig
+        exclude = ("creator",)
```

Comment:
Nitpick: I prefer to use `include` in case we add another field that should stay private, then we're explicit about what fields are included in the form.